Doc/reference/compound_stmts.rst

```diff
@@ -364,10 +364,22 @@ one :keyword:`!except*` clause, the first that matches it. ::
        | ValueError: 1
        +------------------------------------
 
+
 Any remaining exceptions that were not handled by any :keyword:`!except*`
 clause are re-raised at the end, combined into an exception group along with
 all exceptions that were raised from within :keyword:`!except*` clauses.
 
+If the raised exception is not an exception group and its type matches
+one of the :keyword:`!except*` clauses, it is caught and wrapped by an
+exception group with an empty message string. ::
+
+   >>> try:
+   ...     raise BlockingIOError
+   ... except* OSError as e:
```

Comment:
```suggestion
   ... except* BlockingIOError as e:
```
The example is mildly easier to understand if the reader doesn't need to remember that BlockingIOError is a subclass of OSError.